package.json

```diff
@@ -22,11 +22,15 @@
     "@stdlib/math-base-special-ldexp": "^0.0.5",
     "dlv": "^1.1.3",
     "dset": "^3.1.1",
+    "jsrsasign": "^10.8.6",
+    "node-forge": "^1.3.1",
     "tiny-hashes": "^1.0.1"
   },
   "devDependencies": {
     "@types/jest": "^27.4.0",
+    "@types/node": "^16.18.40",
```

Comment:
Do we actually need node types here? It's better to not have any node types in an isomorphic package, since it ensures we don't accidentally use node built-ins in our code (I sometimes add them for tests, but not ideal).